lib/zookeeper.js

```diff
@@ -16,6 +16,12 @@ var zookeeper = require('node-zookeeper-client'),
  */
 var Zookeeper = function (connectionString, options) {
     this.client = zookeeper.createClient(connectionString, options);
+    if (typeof options!='undefined' && typeof options.pathPrefix!='undefined') {

```

Comment:
Looks the new `pathPrefix` option is passed onto zookeeper. How does zookeeper handle that option? Could we move this block up one line and delete the pathPrefix if it was defined?